scripts/generate-rule-files/cli.ts

```diff
@@ -0,0 +1,9 @@
+import { run } from '.';
+
+const selectedPlugins = process.argv
+  .slice(2)
+  .find((arg) => arg.startsWith('--plugins='))
+  ?.replace('--plugins=', '')
+  .split(',');
```

Comment:
We can use `util.parseArgs` https://nodejs.org/api/util.html#utilparseargsconfig

Reply:
Yes we could but it is only with node 18. I didn't want to change the version of node used. So why not, but under control of @Shinigami92

Reply:
Lets do such optimisations in later PRs